src/App/App.js

```diff
@@ -1,22 +1,17 @@
 import { useState, useEffect } from 'react';
 import PropTypes from 'prop-types'
-import movieData from '../sampleData';
 import Header from '../Header/Header'
 import Movies from '../Movies/Movies';
 import MovieDetail from '../MovieDetail/MovieDetail'
 
 import './App.css';
 
 function App() {
-
   const [ movies, setMovies ] = useState([])
-  // const movies = movieData.movies
-  // const [ videos, setVideos ] = useState([])
-  const [selectedMovie, setSelectedMovie] = useState(null);
-  const [selectedVideo, setSelectedVideo] = useState(null);
-  const [currentMovieIndex, setCurrentMovieIndex] = useState(0);
-  const [error, setError ] = useState('')
-
+  const [ selectedMovie, setSelectedMovie ] = useState(null);
```

Comment:
instead of null, consider setting state to the data type of selectedMovie, likely an empty array or object